pkg/asset/internal/templates.go

```diff
@@ -1225,6 +1225,8 @@ spec:
               value: "info"
             - name: CALICO_NETWORKING_BACKEND
               value: "none"
+            - name: CLUSTER_TYPE
+              value: "k8s"
```

Comment:
I think this should be "bootkube", this will indicate the specific integration and also that it is a K8s deployment.

Reply:
If flannel is the only option for networking when using Calico then actually I think it should be "bootkube,canal".

Reply:
Ok, changed to that - bootkube always uses flannel for networking and calico only for policy so canal seems like a good name.

I'm experimenting with calico for networking in [matchbox](https://github.com/coreos/matchbox/pull/645) and [typhoon](https://github.com/poseidon/typhoon), so maybe that becomes an option here in future.